### PR TITLE
#988 - Fix profiler with Ωedit update to v0.9.82

### LINKED ISSUE #988: Data Profiler doesn't work in Windows
See screenshot below on a Windows 11 system:

Screenshot 2024-03-11 111311

### PR DESCRIPTION
**Pull request by scholarsmate**
In Windows running the profiler would kill the Ωedit server.  That behavior was repaired in a recent update to Ωedit.  With this change, the profiler will work when running in Windows.

Fixes #988.

### CODE REVIEW

**Comment by lrbarber** [comment]
@scholarsmate : **All** the automated tests failed?!

**Comment by hdalsania** [comment]
I updated testing checklist for profiler functionality in windows platform.

**Review by hdalsania — APPROVED** [review]
+1 from me

**Comment by shanedell** [comment]
@scholarsmate For future reference, the workflow for merging PRs requires 2 +1's from members of Daffodil Committers/PMC. To know if you have that kind of an approval the checkmark will be green. As seen above my checkmark is green since I am a member of Daffodil Committers/PMC and @hdalsania's checkmark is gray since he is not. We still like not members of the Committers/Members approve and interact with PR's but their approval's don't count toward the required 2 +1's ASF required to merge.

**Pull request merged**
